main/acle.md

```diff
@@ -1427,15 +1435,15 @@ instruction set only. This can be tested for using the following test:
 
 `__ARM_FEATURE_CRC32` is defined to 1 if the CRC32 instructions are
 supported and the intrinsics defined in [CRC32 intrinsics](#crc32-intrinsics)
-are available. These instructions include CRC32B, CRC32H and others.
+are unconditionally usable. These instructions include CRC32B, CRC32H and others.
```

Comment:
> I'm not sure about this kind of change. The statement “feature macro X indicates that Y is available” is still true. I think the difference is that feature macros are no longer the only indication of whether Y is available.

Correct, I'm happy drop these changes from the PR. The Intrinsics section could be extended further. 
 
> Also, #pragma GCC target changes the feature macros (although this doesn't work well when preprocessing the source as a separate step).
> 
> I suppose this is mostly aimed at functions with the target_version attribute. Should we require that, if an implementation supports target_version, and if it recognises a given feature F for the target_version string, it must:
> 
> provide any ACLE header files associated with F
> make the intrinsics for feature F available in a target_version function that enables feature F, if the associated header files have been included.
> (Not suggested wording)

I'll add something like this to the Intrinsics section.

> 
> Perhaps it would help to add a mapping from target_version feature names to feature macros.

unfortunately right now the feature names in the compilers are not standardised and different. 
I'll do a separate PR for this.